packages/cdk/lib/constructs/engines/miniwdl/miniwdl-engine.ts

```diff
@@ -34,6 +35,7 @@ export class MiniWdlEngine extends Engine {
       logGroup: this.logGroup,
       platformCapabilities: [PlatformCapabilities.FARGATE],
       container: {
+        memoryLimitMiB: this.engineMemoryMiB,
```

Comment:
Is there any cost implications out of this change that we need to consider here, looks like something that we could have as a parameter in our context definition.

Reply:
There are although the engine only runs for the lifetime of the workflow so there are not "ongoing costs". Rather than make this configurable I would rather work with Mike Lin to see if we can reduce the need for large memory for workflows with large scatters.